presto-main/src/main/java/com/facebook/presto/sql/planner/LocalExecutionPlanner.java

```diff
@@ -2070,7 +2070,7 @@ private PhysicalOperation createLookupJoin(JoinNode node,
             OptionalInt buildHashChannel = buildHashVariable.map(variableChannelGetter(buildSource))
                     .map(OptionalInt::of).orElse(OptionalInt.empty());
 
-            boolean spillEnabled = isSpillEnabled(context.getSession());
+            boolean spillEnabled = false; //isSpillEnabled(context.getSession());
```

Comment:
What are these changes?

Reply:
Carried in a few changes from my deployment branch, oops